app/models/user/time/Interval.scala

```diff
@@ -17,12 +20,12 @@ case class Month(month: Int, year: Int) extends Interval with Ordered[Month] {
   override def toString = f"$year%d-$month%02d"
 
   def start: Instant =
-    Instant.fromJoda(jodaMonth.dayOfMonth().withMinimumValue().millisOfDay().withMinimumValue())
+    Instant.fromZonedDateTime(asZonedDateTime.`with`(TemporalAdjusters.firstDayOfMonth()).`with`(LocalTime.MIN))
 
   def end: Instant =
-    Instant.fromJoda(jodaMonth.dayOfMonth().withMaximumValue().millisOfDay().withMaximumValue())
+    Instant.fromZonedDateTime(asZonedDateTime.`with`(TemporalAdjusters.lastDayOfMonth()).`with`(LocalTime.MAX))
 
-  private def jodaMonth: DateTime = new DateTime().withYear(year).withMonthOfYear(month)
+  private def asZonedDateTime: ZonedDateTime = ZonedDateTime.of(year, month, 1, 0, 0, 0, 0, ZoneId.systemDefault())
```

Comment:
It’s the server’s local time, so that shouldn’t be a problem. This is the same behavior as before. I’m not opposed to switching to UTC though, since the server’s timezone is somewhat arbitrary, at least for internationally used instances.

Note that this month-grouping only happens for `/users/:id/loggedTime`, which is not used by the frontend, but only by the python client. Not sure if anybody uses that at all at the moment.

Reply:
I think switching to UTC would be good.

Reply:
switched to UTC now. Still uses the ZonedDateTime class to have the fancy TemporalAdjusters available, but uses ZoneId.of("UTC") now.